src/Discord/Builders/CommandAttributes.php

```diff
@@ -33,6 +33,7 @@
  * @property ?string                  $default_member_permissions Set of permissions represented as a bit set.
  * @property bool|null                $dm_permission              Indicates whether the command is available in DMs with the app, only for globally-scoped commands. By default, commands are visible.
  * @property ?bool                    $default_permission         Whether the command is enabled by default when the app is added to a guild. SOON DEPRECATED.
+ * @property bool                     $nsfw                       Indicates whether the command is age-restricted, defaults to `false`
```

Comment:
```suggestion
 * @property bool|null                $nsfw                       Indicates whether the command is age-restricted, defaults to `false`
```

Field is optional as indicated in the docs https://github.com/discord/discord-api-docs/pull/5617/files#diff-eac5037296fa30697ac678095a75beba6d78d1b337263bd2d06d85b147dc616aR29